Extension/src/LanguageServer/client.ts

```diff
@@ -3063,13 +2860,13 @@ export class DefaultClient implements Client {
             return;
         }
 
-        const isCodeAction: boolean = (adjustedLine !== undefined && adjustedLine !== undefined);
-        const initCursorPosition: vscode.Position = isCodeAction ? new vscode.Position(initCursorLine ?? 0, initCursorColumn ?? 0) : editor.selection.active;
+        const isCodeAction: boolean = codeActionArguments !== undefined;
+        const initCursorPosition: vscode.Position = codeActionArguments !== undefined ? new vscode.Position(codeActionArguments.initialCursor.line, codeActionArguments.initialCursor.character) : editor.selection.active;
         const params: GenerateDoxygenCommentParams = {
             uri: editor.document.uri.toString(),
-            position: isCodeAction ? new vscode.Position(adjustedLine ?? 0, adjustedColmn ?? 0) : editor.selection.active,
+            position: (codeActionArguments !== undefined) ? new vscode.Position(codeActionArguments.adjustedCursor.line, codeActionArguments?.adjustedCursor.character) : editor.selection.active,
```

Comment:
This `?.` seems like it should be deleted since it seems invalid -- vscode.Position doesn't accept an `undefined`.